### PR TITLE
feat(es): add benedict_of_nursia and bridget_of_sweden variants

### PR DESCRIPTION
**Pull request by miguelms95**
- benedict_of_nursia_abbot_patron_of_europe
- bridget_of_sweden_religious_copatroness_of_europe

### CODE REVIEW

**Comment by TobiTenno** [comment]
@miguelms95 as the error in the CI mentions, you need to specify what you're adding a "feat" too. in this case, i think you want to reword your commit with `feat(es): add benedict_of_nursia & bridget_of_sweden variants`

**Review by TobiTenno — APPROVED** [review]

File: lib/locales/es.ts [hunk 1]
```diff
@@ -224,6 +224,7 @@ export const locale: Locale = {
     benedict_of_aniane_abbot: 'San Benito de Aniano, Abad',
     benedict_of_jesus_valdivielso_saez_religious: 'San Héctor Valdivielso Sáez, mártir',
     benedict_of_nursia_abbot: 'San Benito, abad, patrono de Europa,',
```
Comment (tukusejssirs) [inline lib/locales/es.ts:226]: There should be no commma at the end of the value. And a patronage title was left here.

```suggestion
    benedict_of_nursia_abbot: 'San Benito, abad',  // src: https://liturgiapapal.org/attachments/article/1093/PROPIO%20DE%20LOS%20SANTOS.pdf#page=9
```

File: lib/locales/es.ts [hunk 2]
```diff
@@ -224,6 +224,7 @@ export const locale: Locale = {
...
+    benedict_of_nursia_abbot_patron_of_europe: 'San Benito, abad, patrono de Europa,',
```
Comment (tukusejssirs) [inline lib/locales/es.ts:227]:
```suggestion
    benedict_of_nursia_abbot_patron_of_europe: 'San Benito, abad, patrono de Europa',  // based on: https://liturgiapapal.org/attachments/article/1093/PROPIO%20DE%20LOS%20SANTOS.pdf#page=9
```

File: lib/locales/es.ts [hunk 3]
```diff
@@ -232,6 +233,7 @@ export const locale: Locale = {
     bonaventure_of_bagnoregio_bishop: 'San Buenaventura, obispo y doctor de la Iglesia',
     boniface_of_mainz_bishop: 'San Bonifacio, obispo y mártir',
     bridget_of_sweden_religious: 'Santa Brígida, religiosa',
+    bridget_of_sweden_religious_copatroness_of_europe: 'Santa Brígida, religiosa, patrona de Europa',
```
Comment (tukusejssirs) [inline lib/locales/es.ts:236]:
```suggestion
     bridget_of_sweden_religious: 'Santa Brígida, religiosa',  // src: https://liturgiapapal.org/attachments/article/1093/PROPIO%20DE%20LOS%20SANTOS.pdf#page=9
    bridget_of_sweden_religious_copatroness_of_europe: 'Santa Brígida, religiosa, patrona de Europa',  // based on: https://liturgiapapal.org/attachments/article/1093/PROPIO%20DE%20LOS%20SANTOS.pdf#page=9
```

**Review by tukusejssirs — CHANGES_REQUESTED** [review]
@miguelms95 and @TobiTenno, sorry, guys, I am still very busy, however, there some issues with the you have merged.

Still I thank you for your time you devote to romcal and the work you do! :pray:

File: lib/locales/es.ts (hunk 3 above)
Comment (TobiTenno) [reply lib/locales/es.ts:236]: sorry about missing these. i'll go put out a new pr if you don't already have one out.

Comment (TobiTenno) [reply lib/locales/es.ts:236]: resolved in #622

File: lib/locales/es.ts (hunk 2 above)
Comment (TobiTenno) [reply lib/locales/es.ts:227]: resolved in #622

File: lib/locales/es.ts (hunk 1 above)
Comment (TobiTenno) [reply lib/locales/es.ts:226]: resolved in #622

**Pull request merged**